pulsar-broker/src/main/java/org/apache/pulsar/broker/service/PulsarCommandSender.java

```diff
@@ -72,10 +72,6 @@ void sendLookupResponse(String brokerServiceUrl, String brokerServiceUrlTls, boo
 
     void sendActiveConsumerChange(long consumerId, boolean isActive);
 
-    void sendSuccess(long requestId);
```

Comment:
This is part of a public API.
Shouldn't it have been deprecated instead of removed ?

Reply:
@cbornet Do we have some plugins or something using this API? I don't aware of it. Maybe we'd better to add 

```
@InterfaceAudience.Public
@InterfaceStability.Stable
```

For the public APIs.